gradle/common/kotlin-common.gradle

```diff
@@ -6,11 +6,17 @@ dependencies {
 }
 
 task sourceJar(type: Jar) {
+    baseName = "${project.parent.name}-${project.name}"
```

Comment:
I see that this thing kind of repeats itself. Do you want to move it somewhere to the upper level?

Reply:
ack